### DISCUSSION TITLE
[Feature] Export photos and videos as in Google Takeout

### DISCUSSION THREAD

**Discussion by FreeWind6** [immich-app/immich#13304] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to be able to export all my photos and videos for transfer, for example, to another storage or for backup. For example, as in Google Takeout

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1644
